### DISCUSSION TITLE
workshop: introduction to FABM

### DISCUSSION THREAD

**Discussion by jornbr** [fabm-model/fabm#58] [Announcements]
Hi all,

On 12 July, we are organizing a half-day workshop on using FABM as part of [the AMEMR conference](https://www.amemr.com/), held in Plymouth. This is introductory workshop oriented towards new users rather than developers; it will focus on running FABM-based biogeochemcial models in common 1D (GOTM) and 3D (global Transport Matrix Method) testbeds. More information can be found here: https://www.amemr.com/workshops1.html.

Best,

Jorn

**Comment by ngdaniel499** [comment]
Hi Jorn,

Is there another workshop coming up soon? I'm particularly interested in running SCHISM with biogeochemical modeling, but unfortunately I missed the session mentioned in the post.

Thanks,
Daniel

**Comment by jornbr** [comment]
Hi Daniel,

We are planning another, longer workshop for spring 2025. More details to follow in September.

Best,

Jorn

**Comment by jornbr** [comment]
The 2025 workshop has now been announced:

https://github.com/fabm-model/fabm/wiki/workshop
